### ISSUE TITLE
leftover Dynamic output in terminal

### ISSUE THREAD

**Issue by wkotlarski** [FlexibleSUSY/FlexibleSUSY#234]
@uukhas, there's still a message
```
Calculate 1-loop corrections
  For rotated fields
  ... processing: Dynamic[DynamicOneLoopNameMM](14/14)
  For tadpoles
      (in progress)
  For gauge eigenstates
    ... processing: Dynamic[DynamicOneLoopNameNM](6/6)
```
in the output. Would it be possible to get rid of it the same way we did with the ones from vertices?

**Comment by uukhas** [comment]
@wkotlarski, thank you for this issue. I think that yes, it is possible. Tell me, please, for which model you get this output? I want to check my fix locally first.

**Comment by wkotlarski** [comment]
I saw it in `U1xMSSM3G` but I guess it's more common.

**Comment by wkotlarski** [comment]
@uukhas, can I close it?

**Comment by uukhas** [comment]
I think that I have fixed it in #236.
